asserts/device_asserts_test.go

```diff
@@ -212,6 +213,7 @@ func (mods *modelSuite) TestModelCheck(c *C) {
 	brandDB := setup3rdPartySigning(c, "brand1", storeDB, db)
 
 	headers := ex.Headers()
+	delete(headers, "authority-id")
```

Comment:
Ditto here and others for comment about why this is deleted. Annoying I know, but I think it will make the tests a bit more readable.